static/assets/scripts/create-devo-json.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Make creating these jsons easier lol. 
+create_json_file() {
+    local input_file="$1"
+    local output_file="$2"
+
+    # Check if input file exists
+    if [ ! -f "$input_file" ]; then
+        echo "Input file '$input_file' does not exist"
+        exit 1
+    fi
+
+    # Prompt user for translation of the text. 
+    read -p "Enter the translation for the book (NOTE: IT DOES NOT SUPPORT MULTIPLE BOOKS IN THE SAME MONTH YET): " replacement_word
```

Comment:
Doesn't have to do it right now, but let's consider supporting multiple books. Should be able to do it via an associative array.